tests/foreman/ui/test_host.py

```diff
@@ -1670,6 +1670,55 @@ def test_global_registration_with_gpg_repo_and_default_package(
     assert repo_url in result.stdout
 
 
+@pytest.mark.tier2
+def test_global_registration_upgrade_subscription_manager(
+    session, module_activation_key, module_os, module_proxy, rhel7_contenthost
```

Comment:
is the purpose here to run only rhel7? otherwise the `rhel_contenthost` with marker 
and e.g. marker `@pytest.mark.rhel_ver_match('[^6].*')`  run all rhel versions except rhel 6 :)

Reply:
I will test that, thank you.

Reply:
@tstrych done that now, thank you